src/main/java/com/hse/knopkabackend/models/KnopkaUser.java

```diff
@@ -0,0 +1,107 @@
+package com.hse.knopkabackend.models;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity(name = "knopka_user")
+@Table(name = "knopka_user")
+public class KnopkaUser {
+    @Id
+    @SequenceGenerator(
+            name = "user_sequence",
+            sequenceName = "user_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "user_sequence"
+    )
+    @Column(
+            name = "id",
+            updatable = false
+    )
+    private Long id;
+
+    @Column(
+            name = "email",
+            nullable = false,
+            unique = true,
+            columnDefinition = "VARCHAR"
```

Comment:
It is better to always limit length by some large number just in case, like VARCHAR(2048)